src/main/resources/application.yml

```diff
@@ -1,11 +1,14 @@
 camel:
-  server-port: 5000
+  server-port: 5002
```

Comment:
check whether this can cause an issue in the cluster server.   If the port number is not overridden from  helm , this should not be modified